go.mod

```diff
@@ -2,46 +2,25 @@ module github.com/keycloak/keycloak-operator
 
 require (
 	cloud.google.com/go v0.57.0 // indirect
-	github.com/Djarvur/go-err113 v0.1.0 // indirect
```

Comment:
Would you mind to remove changes to go.mod and go.sum files?

Reply:
Done now, go.mod and go.sum reverted